tasks/components/layout/WardDisplay.tsx

```diff
@@ -73,7 +73,7 @@ export const WardDisplay = ({
                 })}
                 onTileClick={() => {
                   setOrganizationId(organizationId)
-                  router.push(`/ward/${ward.id}`)
+                  router.push(`/ward/${ward.id}`).then()
```

Comment:
Unsure about those empty `.then()`'s. We could do something like `void router.push()` to prevent a no-floating-promise.
https://stackoverflow.com/questions/55558875/void-before-promise-syntax

Reply:
Will be solved in #1101